src/app/components/homepage/homepage.component.css

```diff
@@ -0,0 +1,34 @@
+.hometile1-text{
```

Comment:
* create a single class called `homeTile` or `tile` - and add all the common css styles within that class
* try and use other methods of positioning the tiles - we have flexbox and grid in css now - using absolute positioning is a bad practice unless you really need to use it
* add a line break between each class within you file

Reply:
Done